app/src/main/java/com/github/se/travelpouch/ui/dashboard/EditActivity.kt

```diff
@@ -52,20 +62,32 @@ import java.util.Locale
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun EditActivity(navigationActions: NavigationActions, activityViewModel: ActivityViewModel) {
+fun EditActivity(navigationActions: NavigationActions, activityViewModel: ActivityViewModel, locationViewModel: LocationViewModel) {
 
   val context = LocalContext.current
 
   val selectedActivity = activityViewModel.selectedActivity.collectAsState()
 
+    val zoneId = ZoneId.of("Europe/Paris")
+    val timeHour = ZonedDateTime.ofInstant(selectedActivity.value!!.date.toDate().toInstant(), zoneId)
```

Comment:
I think there is a problem with the ZoneId. When adding or modifying a time, the wrong time is displayed after saving the activity. It appears with an additional hour

Reply:
I don't have this error, after trying to reproduce the error in different way. I tried to change the zone with default system because it's more appropriate.